northpoint-development/Vector-Tiles-Google-Maps
# BMAPS-1693 Remove vectortiles Point

This ticket was originally for replacing the point library as a dependency, but we no longer need to do this as it was only being used in vector tiles operations that are now imported themself. 

To test this:
1. Pull this branch
2. Open `examples/point-linestring-polygon.html` in your editor
3. Add the beyond maps api key
4. Open the html file in the web browser
5. See that the points are still rendered to the map
